modules/tiles/src/tileset/traversers/i3s-tilset-traverser.js

```diff
@@ -17,7 +17,7 @@ export default class I3STilesetTraverser extends TilesetTraverser {
     return tile._lodJudge === 'DIG';
   }
 
-  // eslint-disable-next-line complexity
+  /* eslint-disable max-depth */
```

Comment:
Nit: We use `eslint-disable-next-line` instead of turning off warnings for the rest of the file.

Reply:
I've added `/* eslint-enable max-depth */` right after this method

Reply:
Yes fine, that also works. 

Just pointing out that the `eslint-disable-next-line` style what we try use in the rest of the loaders code base. We originally used `enable/disable` but have been replacing them with the per-line style.

Reply:
Done